### DISCUSSION TITLE
renovate web app ?

### DISCUSSION THREAD

**Discussion by omercnet** [mend/renovate-ce-ee#453] [Q&A] [answered]
is there a community version of https://developer.mend.io/ to use?

**Comment by rarkins** [comment]
No, not yet - it requires a lot more complexity than the single container we have today! Log storage, DB, web server, etc. We are exploring options though

**Reply by omercnet** [reply]
thanks for even considering it, really appreciated!

a simple stateless version that just reflects what the current renovate-ce/ee writes to the db will be a great start to provide visibility in to the current state of renovate

**Comment by omercnet** [comment]
with the reporting/logs/jobs apis - a simple app showing job status and logs would be super valuable 
any plans on building that?

**Reply by rarkins** [reply]
Enterprise features are likely to go in the enterprise edition if/once built

**Reply by omercnet** [reply]
I see most useful features like job status, logs and adding new tasks are in CE

**Reply by justo-mend** [reply]
Thanks @omercnet. That's good to know.

**Comment by jamietanna** [comment, marked as answer]
Web UI is available as of [Enterprise Self-Hosted 13.6.0](https://github.com/mend/renovate-ce-ee/releases/tag/13.6.0)
